redisinsight/ui/src/utils/monaco/regex.ts

```diff
@@ -0,0 +1,2 @@
+// Escape special regex characters in tokens
+export const sanitizeToken = (token: string = '') => token.replace(/[.*+?^${}()|[\]\\]/g, '\\$&') || ''
```

Comment:
I'm thinking about adding a bunch of simple tests to act as a safety net in the future, since nobody understands regex 😀